ReplayMessenger.device.lib.nut

```diff
@@ -108,38 +136,54 @@ class ReplayMessenger {
     }
 
     function _onAck(message) {
-        _log("ACKed message name: '" + message.payload.name + "', data: " + message.payload.data);
-        if ("addr" in message.metadata && message.metadata.addr) {
-            local addr = message.metadata.addr;
-            local logger = _spiFL[message.metadata.loggerName];
-            _log("Erasing object at address: " + addr);
-
-            logger.erase(addr);
-            message.metadata.addr = null;
-            _scheduleRetryIfConnected();
+        if ("metadata" in message && "loggerName" in message.metadata) {
```

Comment:
Just a minor cosmetic comment here:

I would extract this into a separate method:

```
function _isRecognized(message) {
    return "metadata" in message && "loggerName" in message.metadata;
}
```
and do a simple check here:

```
    if (! _isRecognized(message)) {
        return; // skip all non replay messenger
    }
```